dbms/src/AggregateFunctions/AggregateFunctionGroupBitmapData.h

```diff
@@ -103,9 +104,10 @@ class RoaringBitmapWithSmallSet : private boost::noncopyable
 
         if (isLarge())
         {
-            UInt32 cardinality = roaring_bitmap_get_cardinality(rb);
-            writePODBinary(cardinality, out);
-            db_ra_to_uint32_array(out, &rb->high_low_container);
+            uint32_t expectedsize = roaring_bitmap_portable_size_in_bytes(rb);
+            std::string s(expectedsize,0);
+            roaring_bitmap_portable_serialize(rb, const_cast<char*>(s.data()));
+            writeStringBinary(s,out);
```

Comment:
Is that change backwards compatible? I.e. data written by new variant of code can be read by old variant and vice versa?

Reply:
It's mentioned in PR description.